File: Part 2/C. Board Moves.cpp
// Solver : Nahin the Coder
// Date : 08 - 08 - 2021

#include<bits/stdc++.h>
#define ll long long int
using namespace std;

int main()
{
    ll test;
    cin>>test;
    while(test--)
    {
    	ll n;
    	ll counter = 1;
    	ll xy = 8;
    	ll sum = 0;
    	cin>>n;
    	if(n == 1)
    	{
    		cout<<0<<endl;
    		continue;
		}
    	for(ll i = 3; i<=n; i = i+2)
    	{
    		sum = sum + xy*counter;
    		counter++;
    		xy = 8*counter;
		}
		cout<<sum<<endl;
    }
    return 0;
}

File: Solved by Python/C. Board Moves.py
# Nahin the Coder
# Date : 07 - 08 - 2021


testcase = int(input())
while testcase > 0:
    n = int(input())
    counter = int(1)
    xy = int(8)
    result = int(0)
    if n == 1:
        print(0)
        testcase = testcase - 1
        continue
    else:
        i = int(3)
        while i <= n:
            result = result + xy*counter
            counter = counter + 1
            xy = 8*counter
            i = i+2
        print(result)
    testcase = testcase - 1
